homeassistant/components/deconz/cover.py

```diff
@@ -22,6 +22,12 @@
 from .deconz_device import DeconzDevice
 from .gateway import get_gateway_from_config_entry
 
+DEVICE_CLASS = {
+    DAMPERS[0]: DEVICE_CLASS_DAMPER,
+    WINDOW_COVERS[0]: DEVICE_CLASS_SHADE,
+    WINDOW_COVERS[1]: DEVICE_CLASS_SHADE,
+}
```

Comment:
Really honest... I don't think this made things clearer over the old property method :)

Reply:
Yes I know :) I'll revisit it.